static/app/components/globalSdkUpdateAlert.tsx

```diff
@@ -45,21 +45,21 @@ const recordAnalyticsSeen = ({organization}: AnalyticsOpts) =>
   trackAnalyticsEvent({
     eventKey: 'sdk_updates.seen',
     eventName: 'SDK Updates: Seen',
-    organizationId: organization.id,
+    organization_id: organization.id,
```

Comment:
shouldn't this be
```suggestion
    org_id: organization.id,
```

based on what's in reload?
https://github.com/getsentry/reload/blob/master/reload_app/events.py#L988-L996

Reply:
This is kind of dumb but.. https://github.com/getsentry/getsentry/blob/e4e77129fc2e04ad9fdf93b496ba98bb852c750b/static/getsentry/gsApp/utils/trackAnalyticsEvent.tsx#L81-L83

Reply:
🙈 welp...